kite-data/kite-data-spark/src/main/scala/org/kitesdk/data/spark/KiteDatasetReader.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2015 Cloudera, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kitesdk.data.spark
+
+import org.apache.hadoop.mapreduce.Job
+import org.apache.spark.sql.{DataFrame, SQLContext}
+import org.kitesdk.data.Dataset
+
+object KiteDatasetReader {
+
+  def readAsDataFrame(sqlContext: SQLContext,
```

Comment:
Naming: should this be KiteDatasetLoader and loadAsDataFrame, for symmetry with KiteDatasetSaver/saveAsKiteDataset.